pkg/cli/private_key_import_cfg.go

```diff
@@ -89,18 +91,40 @@ func (c *PrivateKeyImportCommand) Run(ctx context.Context, args []string) error
 	if err != nil {
 		return fmt.Errorf("encountered error when creating/getting key ring: %w", err)
 	}
-	logger.InfoContext(ctx, "Got key ring successfully\n", "key_ring", gotKeyRing.GetName())
+	logger.DebugContext(ctx, "Got key ring successfully\n", "key_ring", gotKeyRing.GetName())
 	gotKey, err := keyServer.GetOrCreateKey(ctx, c.cfg.ProjectID, c.cfg.Location, c.cfg.KeyRing, c.cfg.Key)
 	if err != nil {
 		return fmt.Errorf("encountered error when creating/getting key: %w", err)
 	}
-	logger.InfoContext(ctx, "Got key successfully\n", "key_ring", gotKey.GetName())
+	logger.DebugContext(ctx, "Got key successfully", "key_ring", gotKey.GetName())
 	gotImportJob, err := keyServer.GetOrCreateImportJob(ctx, c.cfg.ProjectID, c.cfg.Location, c.cfg.KeyRing, c.cfg.ImportJobPrefix)
 	if err != nil {
 		return fmt.Errorf("encountered error when creating/getting import job: %w", err)
 	}
-	logger.InfoContext(ctx, "Got import job successfully\n", "import_job", gotImportJob.GetName())
-	// TODO import key version
+	logger.DebugContext(ctx, "Got import job successfully", "import_job", gotImportJob.GetName())
+	importedJob, err := keyServer.GetImportJob(ctx, gotImportJob.GetName())
+	if err != nil {
+		return fmt.Errorf("encountered error when checking state of import job: %w", err)
+	}
+	if importedJob.GetState() != kmspb.ImportJob_ACTIVE {
+		return fmt.Errorf("import job is not in active stage, you may re-run it to retry")
+	}
+	createdKeyVersion, err := keyServer.ImportManuallyWrappedKey(ctx, gotImportJob.GetName(), gotKey.GetName(), c.cfg.PrivateKey)
+	if err != nil {
+		return fmt.Errorf("failed to import key version: %w", err)
+	}
+	logger.DebugContext(ctx, "Got key version imported", "key_version", createdKeyVersion.GetName())
+	//sleep 3 second to wait for the key version to becoming enabled.
```

Comment:
I think we want to do this with go-retry and wait for the key to get into the state we want instead of an arbitrary sleep time.

Reply:
fixed with `go-retry`